search-testutils/build.gradle

```diff
@@ -0,0 +1,65 @@
+dependencies {
+
+    compileOnly project(':search-commons')
+    testImplementation project(':search-commons')
+
+    implementation group: 'org.hamcrest', name: 'hamcrest', version: '2.2'
+    implementation group: 'org.mockito', name: 'mockito-core', version: '3.5.7'
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-api', version: junit5Version
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-params', version: junit5Version
+    implementation group: 'org.junit.jupiter', name: 'junit-jupiter-engine', version: junit5Version
+
+
+}
+
+//dependencies {
```

Comment:
Why is this here?

Reply:
..don't copy code, and leave it... Gone!